portfolio/src/main/webapp/index.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="UTF-8">
     <title>Tallie's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCA1CSIx2Pn8K4N2gXjMHSIJxvZIKXYk8U"></script>
```

Comment:
Beware with including keys in the publicly facing HTML code.

Reply:
When I deploy the project - what would be the best way to include the API key without making it publicly visible?

Reply:
It seems that Google Maps actually require including the key in JS.

The solution to avoid misuse of the key is restricting the referral that can use the API key: https://developers.google.com/maps/documentation/javascript/get-api-key#restrict_key

I would apply the restrictions to that key, as it has already been published, and it could be misused by third parties.